MAUI/DataForm/Working-with-DataForm.md

```diff
@@ -0,0 +1,575 @@
+---
+layout: post
+title: Working with DataForm in Maui DataForm control | Syncfusion
```

Comment:
Working with DataForm in .Net MAUI DataForm control | Syncfusion

Reply:
title changed